translations/mod-fqm-manager/en.json

```diff
@@ -231,6 +231,7 @@
   "entityType.composite_instances.instance": "Instance",
   "entityType.composite_instances.inst_stat": "Instance status",
   "entityType.composite_instances.mode_of_issuance": "Mode of issuance",
+  "entityType.composite_instances.instance_type": "Resource type",
```

Comment:
Technically, this doesn't exactly follow the requirements laid out in MODFQMMGR-309 (which is just asking for the instance type name, and calling _that_ "Resource type"), but I figured that it's probably better to give that name to the whole child ET